symbol_exporter/ast_symbol_extractor.py

```diff
@@ -193,6 +195,21 @@ def _add_symbol_to_star_imports(self, imported_symbol):
         default = dict(type=SymbolType.STAR_IMPORT, data=dict(imports=set()))
         self.symbols.setdefault("*", default)["data"]["imports"].add(imported_symbol)
 
+    def post_process_symbols(self):
```

Comment:
Why not make symbols a property and let that be the only entry point into the results?